tests/spread/plugins/v2/snaps/colcon-stage-snaps/snap/snapcraft.yaml

```diff
@@ -0,0 +1,61 @@
+name: colcon-stage-snaps
```

Comment:
Can this be tested with the extension? Looks like the snapcraft.yaml is pre-expanded.  Or is this setup not supported with the extension?

Reply:
My bad, I was toying with the extension and forgot to revert that. Will fix it.